test/test_ecco4_en4.jl

```diff
@@ -194,45 +172,45 @@ end
                                         southern = (φ₁, φ₂),
                                                z = (z₁, 0))
 
-        T_restoring = ECCORestoring(:temperature, arch; start_date, end_date, mask, inpainting, rate=1/1000)
-
-        fill!(T_restoring.field_time_series[1], 1.0)
-        fill!(T_restoring.field_time_series[2], 1.0)
-
-        T = CenterField(grid)
-        fields = (; T)
-        clock  = Clock(; time = 0)
-
-        @test @allowscalar T_restoring(1, 1,   10, grid, clock, fields) == T_restoring.rate
-        @test @allowscalar T_restoring(1, 11,  10, grid, clock, fields) == T_restoring.rate / 2
-        @test @allowscalar T_restoring(1, 21,  10, grid, clock, fields) == 0
-        @test @allowscalar T_restoring(1, 80,  10, grid, clock, fields) == 0
-        @test @allowscalar T_restoring(1, 90,  10, grid, clock, fields) == T_restoring.rate / 2
-        @test @allowscalar T_restoring(1, 100, 10, grid, clock, fields) == T_restoring.rate
-        @test @allowscalar T_restoring(1, 1,   5,  grid, clock, fields) == 0
-        @test @allowscalar T_restoring(1, 10,  5,  grid, clock, fields) == 0
+        for name in (:temperature, :salinity)
+            var_restoring = DatasetRestoring(name; dataset, start_date, end_date, mask, inpainting, rate=1/1000)
```

Comment:
Here as well, since the previous `LatitudeLongitudeGrid` accepts an `arch`, we also might want to do
```suggestion
            var_restoring = DatasetRestoring(name, arch; dataset, start_date, end_date, mask, inpainting, rate=1/1000)
```
here